news/templatetags/news.py

```diff
@@ -20,25 +20,23 @@ def user_news(context, num, more=0):
     news = context['course_news']
 
     if context['is_course_staff']:
-        alerts,news = news.for_staff()
+        news = news.for_staff()
     else:
         user = context['request'].user
-        alerts,news = news.for_user(
+        news = news.for_user(
             not user.is_authenticated
             or user.userprofile.is_external
         )
 
     i = 0
     for item in news:
```

Comment:
This should use `enumerate`, but as this loop is useless now, it could be removed. The other line could be written as

```python
if more > 0 and more < len(news):
    news[more]['begin_more'] = True
```

Reply:
This was also simplified here: https://github.com/apluslms/a-plus/pull/570/commits/03b9cecbba023f49826f94f3460bcaab1e449c2c.